serverless/reference/kn-event-ref.adoc

```diff
@@ -0,0 +1,12 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="kn-event-ref"]
+= kn event
+:context: kn-event-ref
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+You can use the `kn event` set of commands for managing cloud events.
```

Comment:
Here I probably would describe it a little different, once the phrase as is may lead me think I can do anything about events like not just produce, but view, list, change or consume events.

So what about...
 
```suggestion
You can use the `kn event` set of commands for building and sending cloud events from the command line.
```

Reply:
@jrangelramos I used "managing" in case `kn event` gets new functionality in the future. (I didn't want to forget to update the introduction).
@cardil Do you expect that `kn event` will get additional functionality beside `build` and `send`?

Reply:
Yes. We anticipate additional commands and functionality to be incorporated to `kn event` in the future :crossed_fingers: 

Here's a doc if you are interested: https://docs.google.com/document/d/11luj2wqajEsiaNXHjnGNBgc5viHdObCpqEwCAYnYbE8

Reply:
@cardil Thanks. @jrangelramos I've used your " from the command line." at the end :+1: